src/renderer/src/routes/modals/PrepareHardwareWalletMigrationNeo3/PrepareHardwareWalletContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TbArrowRight } from 'react-icons/tb'
+import { Button } from '@renderer/components/Button'
+import { Stepper } from '@renderer/components/Stepper'
+
+type TProps = {
+  currentStep: number
+  nextLabel: string
+  canNext: boolean
+  isNexting?: boolean
+  onNext: () => void
```

Comment:
Change this props to buttonProps: TButtonProps. You will need to export the TButtonProps